packages/block-library/src/page-list/index.php

```diff
@@ -146,27 +146,37 @@ function nest_pages( $current_level, $children ) {
  * @return string Returns the page list markup.
  */
 function render_block_core_page_list( $attributes, $content, $block ) {
+	global $post;
```

Comment:
One way of avoiding global, would be storing a `parent` attribute rather than `childrenof` boolean.

We can still show a boolean toggle in the block settings, but we'd internally serialize the parent post id.

This would match the query parameter option of the pages api https://developer.wordpress.org/rest-api/reference/pages/

<img width="770" alt="Screen Shot 2021-05-17 at 8 54 22 AM" src="https://user-images.githubusercontent.com/1270189/118519095-a5cb4a80-b6ed-11eb-9802-64bbec88f18e.png">

Reply:
I don't think this will work either because it will fix the menu to a specific page, the same issue highlighted in the video above. It won't let a single template be used for multiple pages with children pages, which is the use case I'm aiming to solve in this PR.

The following pages:
https://mkaz.blog/working-with-vim
https://mkaz.blog/working-with-rust
https://mkaz.blog/working-with-go

All use the same template and the menu is created from the child pages of the parent.